presto-main/src/test/java/io/prestosql/sql/TestExpressionInterpreter.java

```diff
@@ -455,6 +455,9 @@ public void testIn()
         assertOptimizedEquals("bound_decimal_short in (123.455, 123.46, 123.45)", "true");
         assertOptimizedEquals("bound_decimal_long in (12345678901234567890.123, 9876543210.9874561204, 9876543210.98745612035)", "true");
         assertOptimizedEquals("bound_decimal_long in (9876543210.9874561204, null, 9876543210.98745612035)", "null");
+
+        assertOptimizedEquals("unbound_integer in (1)", "unbound_integer = 1");
+        assertOptimizedEquals("unbound_integer in (unbound_integer)", "unbound_integer = unbound_integer");
```

Comment:
ideally this should roll up to `unbound_integer IS NOT NULL` (for types other than floating point)

maybe you could also have a test with `var IN (some_other_var)`?

Reply:
btw `in` should be uppercase

Reply:
> ideally this should roll up to unbound_integer IS NOT NULL (for types other than floating point)

I knew you were going to say that :)

Yeah, we should add another optimization for that at some point, but is seems like a pretty uncommon thing to occur in real queries.

Reply:
Right, i don't expect you do add this other optimization now. I'm just saying that, from test perspective, it's slightly better to have two cases (same var, different vars).